src/explore-education-statistics-admin/src/pages/publication/components/LegacyReleasesTable.tsx

```diff
@@ -41,9 +44,15 @@ const getModalTitle = (confirmAction?: ConfirmAction): string => {
 };
 
 interface Props {
+  createRoute?: string; // EES-3217 remove when live
+  editRoute?: (id: string) => string; // EES-3217 remove when live
   publication: MyPublication;
 }
-const LegacyReleasesTable = ({ publication }: Props) => {
+const LegacyReleasesTable = ({
+  createRoute,
+  editRoute,
+  publication,
```

Comment:
Would be good to decouple the dependency on `publication` as we end up having to pass in a full `MyPublication` when we only need `legacyReleases` and a `publicationId`.

The upcoming backend work in [EES-3575](https://dfedigital.atlassian.net/browse/EES-3575) will split the list of legacy releases out into a separate endpoint so we should save ourselves the hassle of separating this out later.

This might also simplify some of the tests as we won't need to setup some of the repetitive test data.